project/Build.scala

```diff
@@ -1440,6 +1459,73 @@ object Build {
       settings(commonBenchmarkSettings).
       enablePlugins(JmhPlugin)
 
+    def asScala3doc: Project = {
+      def generateDocumentation(targets: String, name: String, outDir: String, params: String = "") = Def.taskDyn {
+          val sourceMapping = "=https://github.com/lampepfl/dotty/tree/master#L"
+          run.in(Compile).toTask(s""" -o output/$outDir -t $targets -n "$name" -s $sourceMapping $params""")
+      }
+
+      project.settings(commonBootstrappedSettings).
+        dependsOn(`scala3-compiler-bootstrapped`).
+        dependsOn(`scala3-tasty-inspector`).
+        settings(
+          resolvers += Resolver.jcenterRepo,
+          resolvers += Resolver.bintrayRepo("kotlin", "kotlin-dev"),
+          resolvers += Resolver.bintrayRepo("virtuslab", "dokka"),
+          libraryDependencies ++= Seq(
+            "org.scala-lang" %% "scala3-tasty-inspector" % scalaVersion.value,
```

Comment:
For internal dependency we should use `dependsOn` (and in fact there is ``dependsOn(`scala3-tasty-inspector`)`` above so this line can just be removed)

Reply:
ah, we've forgot to remove that one during migration, will fix that